src/index.js

```diff
@@ -26,8 +27,18 @@ module.exports = {
     serialize: (data, cb) => {
       cb(null, data)
     },
-    cid: (data, cb) => {
-      cb(null, new CID(1, 'raw', data))
+    cid: (data, options, cb) => {
+      if (options instanceof Function) {
```

Comment:
Searching through the existing IPFS/IPLD code base it seems that the preferred way for checking if something is a function is: ` if (typeof options !== 'function') {`. My guess is that it is related to code minifiers. Please change it.